### PR TITLE
Skipping backend devices

### PR DESCRIPTION
**Pull request by mgornikov**
Devices that used as backed for logical layers (dmraid, bcache, LVM , etc) have "masters" in Ansible "devices" fact. This patch skip such devices on initial lookup.

### CODE REVIEW

File: group_vars/rhcs.yml.sample [hunk 1]
```diff
@@ -29,7 +29,7 @@ dummy:
 #  mimic: 13
 
 # Directory to fetch cluster fsid, keys etc...
-fetch_directory: ~/ceph-ansible-keys
+#fetch_directory: fetch/
```
Comment (leseb) [inline group_vars/rhcs.yml.sample]: why did you change that?

Comment (mgornikov) [reply group_vars/rhcs.yml.sample]: It was generated when run ./generate_group_vars_sample.sh

Comment (guits) [reply group_vars/rhcs.yml.sample]: @mgornikov probably due to a bad rebase on master

**Comment by leseb** [comment]
Closing this due to inactivity, feel free to re-open, thanks!
